### DISCUSSION TITLE
stesso indirizzo IP a wlan0 ed eth0 su raspi4

### DISCUSSION THREAD

**Discussion by poliant** [indomus/forum#165] [Networking]
ciao a tutti,
sul mio raspi4 dove gira HA vorrei impostare lo stesso IP statico sia alla wlan0 che alla eth0 così che quando faccio il NAT/PAT dal router posso accedere ad HA dall'esterno con la stessa porta (e.g. 8124 o similare)

Sapete indirizzarmi?

grazie

**Comment by Scialla** [comment]
tecnicamente è possibile ma altamente sconsigliabile, è probabile avere poi un sacco di problemi nei pacchetti trasmessi. 
Googla con "wlan eth same ip" così ti rendi conto delle spiegazioni tecniche date alla stessa domanda posta da altre persone
